source/Interface.cpp

```diff
@@ -654,8 +654,8 @@ Interface::BarElement::BarElement(const DataNode &node, const Point &globalAncho
 	Load(node, globalAnchor);
 
 	// Fill in a default color if none is specified.
-	if(!color)
-		color = GameData::Colors().Get("active");
+	if(!fromColor)
+		fromColor = toColor = GameData::Colors().Get("active");
```

Comment:
hmm I'm not sure
```suggestion
	{
		fromColor = GameData::Colors().Get("active");
		toColor = fromColor;
	}
```

Reply:
Chaining assignments in C/++ is well defined, and is a more succinct way of doing this.

Reply:
Eh, this is one side effect, just happens to assign to two variables at once.